websites/W/World Romance Translation/presence.ts

```diff
@@ -17,11 +17,18 @@ presence.on("UpdateData", async () => {
     if (searchQuery === null) presenceData.details = "Browsing Home Page";
```

Comment:
then change this english text there too?

Reply:
i have changed it